go/stellar/stellarsvc/service_test.go

```diff
@@ -1430,6 +1431,7 @@ func setupTestsWithSettings(t *testing.T, settings []usetting) ([]*TestContext,
 		switch setting {
 		case usettingFull:
 		case usettingMobile:
+			os.Setenv("KEYBASE_APP_TYPE", string(libkb.MobileAppType))
```

Comment:
I think there has to be a way not to do this. Maybe @patrickxb knows?

Reply:
please

Reply:
won't this break other tests that run at the same time

Reply:
it's definitely possible, though i don't think it would in this specific case. anyway, i figured out another way to do it. it's a little different from how the rest of the app does it, but i think that's ok in this case.